.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         pip install -r ${{ github.workspace }}/src/promptflow/dev_requirements.txt
         gci ./promptflow -Recurse | % {if ($_.Name.Contains('.whl')) {python -m pip install "$($_.FullName)"}}
         gci ./promptflow-tools -Recurse | % {if ($_.Name.Contains('.whl')) {python -m pip install $_.FullName}}
+        pip install openai==0.28.0
```

Comment:
add comment to explain why need to pin and share our plan to remove the pin as todo.

Reply:
added.